west.yml

```diff
@@ -313,7 +313,7 @@ manifest:
       revision: fb00b383072518c918e2258b0916c996f2d4eebe
       path: modules/lib/loramac-node
     - name: lvgl
-      revision: b03edc8e6282a963cd312cd0b409eb5ce263ea75
+      revision: pull/67/head
```

Comment:
I intended to open a PR upstream to incorporate this, but from what I can see this should be resolved now:
https://github.com/zephyrproject-rtos/lvgl/pull/67#issuecomment-3595068190

So as soon as the module is updated this might be dropped as well.